repo/packages/D/dcos-ui/6/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "4.0",
+  "name": "dcos-ui",
+  "version": "2.100.0",
```

Comment:
we need a `2.x` version here that's bigger than `2.83.x` which is the version 1.13.7 shipped with because the update-service-ui will only show the update-button if the major version matches. so we introduce an artificial `2.100.0` here.

we'll soon be able to get rid of all versions `< 5.x`, as `5.x` will soon be capable of supporting all supported versions of dcos.